apps/console/src/features/identity-providers/components/forms/authenticators/google-authenticator-form.tsx

```diff
@@ -232,22 +232,22 @@ export const GoogleAuthenticatorForm: FunctionComponent<GoogleAuthenticatorFormP
         setInitialValues(resolvedInitialValues);
     }, [ originalInitialValues ]);
 
-    /**
-     * Checking the current user is a super user or not.
-     *
-     * @returns Is super user logged in.
-     */
-    const isSuperTenantLogin = (): boolean => {
-        return AppConstants.getTenant() === AppConstants.getSuperTenant();
-    };
-
     /**
      * Checking ability to enable Google One Tap.
      *
      * @returns Whether enable Google One Tap or not.
      */
     const isEnableGoogleOneTap = (): boolean => {
-        return !isGOTEnabledForSuperTenantOnly || isSuperTenantLogin();
+        if (googleOneTapEnabledTenants?.length > 0) {
+            return googleOneTapEnabledTenants.includes(AppConstants.getTenant());
+        }
+
+        /**
+         * To enable Google One Tap for all tenants
+         * if this configuration is not defined or empty.
+         */
+
```

Comment:
```suggestion
```

Reply:
This is fixed 90114ebb54a8820fdf7882291874775c7d6becc0